Configuration/Generator/python/Upsilon1SToMuMu_forSTEAM_14TeV_TuneCP5_cfi.py

```diff
@@ -0,0 +1,52 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from Configuration.Generator.Pythia8BottomoniumSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
+                         pythiaPylistVerbosity = cms.untracked.int32(0),
+                         filterEfficiency = cms.untracked.double(0.53),
+                         pythiaHepMCVerbosity = cms.untracked.bool(False),
+                         crossSection = cms.untracked.double(9090000.0),
+                         comEnergy = cms.double(14000.0),
+                         maxEventsToPrint = cms.untracked.int32(0),
+                         PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
+	pythia8BottomoniumSettingsBlock,
```

Comment:
probably you will need to fix the indentation here otherwise python will complain

Reply:
fixed